workloads/sdn2ovn/run.sh

```diff
@@ -31,51 +37,115 @@ _check_pods(){
     exit 1    
 }
 
+_index_results(){
+    METADATA=$(cat <<EOF
+{
+  "uuid": "${UUID}",
+  "cluster_name": "$(oc get infrastructure.config.openshift.io cluster -o json 2>/dev/null | jq -r .status.infrastructureName)",
+  "before_network_type": "$3",
+  "after_network_type": "$4",
+  "network_migration_duration": "$1",
+  "mcp_duration": "$2",
+  "master_count": "$(oc get node -l node-role.kubernetes.io/master= --no-headers 2>/dev/null | wc -l)",
+  "worker_count": "$(oc get node --no-headers -l node-role.kubernetes.io/workload!="",node-role.kubernetes.io/infra!="",node-role.kubernetes.io/worker="" 2>/dev/null | wc -l)",
+  "infra_count": "$(oc get node -l node-role.kubernetes.io/infra= --no-headers --ignore-not-found 2>/dev/null | wc -l)",
+  "total_node_count": "$(oc get nodes --no-headers 2>/dev/null | wc -l)",
+  "ocp_version": "$5",
+  "timestamp": "$(date +%s%3N)"
+}
+EOF
+)
+    printf "Indexing installation timings to ${ES_SERVER}/${_es_index}"
+    curl -k -sS -X POST -H "Content-type: application/json" ${ES_SERVER}/${_es_index}/_doc -d "${METADATA}" -o /dev/null
+    return 0
+}
+
 echo "Migration of cluster default CNI to OVNKubernetes"
 
-N_TYPE=$(oc get Network.operator.openshift.io cluster  -o json | jq -r '.spec.defaultNetwork.type')
+echo "Take back up of cluster network configuration"
+oc get Network.config.openshift.io cluster -o yaml > $TEMP_DIR/cluster-openshift-sdn.yaml
+BEFORE_N_TYPE=$(oc get Network.operator.openshift.io cluster  -o json | jq -r '.spec.defaultNetwork.type')
 echo "Current CNI is $N_TYPE"
```

Comment:
Typo here. N_TYPE should have been replaced with BEFORE_N_TYPE

Reply:
I have updated - ptal!